src/vs/workbench/services/editor/browser/codeEditorService.ts

```diff
@@ -75,7 +77,9 @@ export class CodeEditorService extends CodeEditorServiceImpl {
 		// should be pinned or not. This ensures that the source of a navigation
 		// is not being replaced by the target. An example is "Goto definition"
 		// that otherwise would replace the editor everytime the user navigates.
+		const pinPreview = this.configurationService.getValue<boolean>('workbench.editor.enablePreviewFromCodeNavigation');
```

Comment:
I would prefer if we would lookup this setting similar to how we do it here:

https://github.com/microsoft/vscode/blob/4031280b29ee6e6d85abac0da91167440c575175/src/vs/workbench/contrib/codeEditor/browser/quickaccess/gotoLineQuickAccess.ts#L33-L33

Aka add it to `IWorkbenchEditorConfiguration`

Reply:
changed it, thanks for the pointer